### ISSUE TITLE
你好 neural_style.py: error: the following arguments are required: --style_imgs

### ISSUE THREAD

**Issue by yingzhongkong** [anishathalye/neural-style#184]
usage: neural_style.py [-h] [--verbose] [--img_name IMG_NAME] --style_imgs
                       STYLE_IMGS [STYLE_IMGS ...]
                       [--style_imgs_weights STYLE_IMGS_WEIGHTS [STYLE_IMGS_WEIGHTS ...]]
                       [--content_img CONTENT_IMG]
                       [--style_imgs_dir STYLE_IMGS_DIR]
                       [--content_img_dir CONTENT_IMG_DIR]
                       [--init_img_type {random,content,style}]
                       [--max_size MAX_SIZE] [--content_weight CONTENT_WEIGHT]
                       [--style_weight STYLE_WEIGHT] [--tv_weight TV_WEIGHT]
                       [--temporal_weight TEMPORAL_WEIGHT]
                       [--content_loss_function {1,2,3}]
                       [--content_layers CONTENT_LAYERS [CONTENT_LAYERS ...]]
                       [--style_layers STYLE_LAYERS [STYLE_LAYERS ...]]
                       [--content_layer_weights CONTENT_LAYER_WEIGHTS [CONTENT_LAYER_WEIGHTS ...]]
                       [--style_layer_weights STYLE_LAYER_WEIGHTS [STYLE_LAYER_WEIGHTS ...]]
                       [--original_colors]
                       [--color_convert_type {yuv,ycrcb,luv,lab}]
                       [--color_convert_time {after,before}] [--style_mask]
                       [--style_mask_imgs STYLE_MASK_IMGS [STYLE_MASK_IMGS ...]]
                       [--noise_ratio NOISE_RATIO] [--seed SEED]
                       [--model_weights MODEL_WEIGHTS]
                       [--pooling_type {avg,max}] [--device {/gpu:0,/cpu:0}]
                       [--img_output_dir IMG_OUTPUT_DIR]
                       [--optimizer {lbfgs,adam}]
                       [--learning_rate LEARNING_RATE]
                       [--max_iterations MAX_ITERATIONS]
                       [--print_iterations PRINT_ITERATIONS] [--video]
                       [--start_frame START_FRAME] [--end_frame END_FRAME]
                       [--first_frame_type {random,content,style}]
                       [--init_frame_type {prev_warped,prev,random,content,style}]
                       [--video_input_dir VIDEO_INPUT_DIR]
                       [--video_output_dir VIDEO_OUTPUT_DIR]
                       [--content_frame_frmt CONTENT_FRAME_FRMT]
                       [--backward_optical_flow_frmt BACKWARD_OPTICAL_FLOW_FRMT]
                       [--forward_optical_flow_frmt FORWARD_OPTICAL_FLOW_FRMT]
                       [--content_weights_frmt CONTENT_WEIGHTS_FRMT]
                       [--prev_frame_indices PREV_FRAME_INDICES [PREV_FRAME_INDICES ...]]
                       [--first_frame_iterations FIRST_FRAME_ITERATIONS]
                       [--frame_iterations FRAME_ITERATIONS]
neural_style.py: error: the following arguments are required: --style_imgs
我应该怎么做 what should I do？

**Comment by anishathalye** [comment]
My neural-style doesn't have a `style_imgs` argument. Are you using a fork, or some other neural style implementation?
